app/common/controllers/intercom.js

```diff
@@ -0,0 +1,28 @@
+module.exports = [
+    '$scope',
+    '$rootScope',
+    'UserEndpoint',
+function (
+    $scope,
+    $rootScope,
+    UserEndpoint
+) {
+    $rootScope.$on('event:authentication:login:succeeded', function () {
```

Comment:
Can we trigger some kind of 'event' in intercom to record the login/logout for each user?